### ISSUE TITLE
[Feature request] Customize lighting angle

### ISSUE THREAD

**Issue by JL102** [emilianavt/VSeeFaceReleases#40]
I would greatly appreciate the ability to change the angle of the light being cast on my character. I did a livestream with a friend who had a PNGtuber, and in order to make my character's lighting vaguely match her art's lighting, she had to horizontally flip my video stream. But still, it wasn't the greatest match.

I was going to try and contribute this feature myself, but I was disappointed to find that VSeeFace is closed source, so I can't contribute this feature. I would greatly appreciate if you would be able to add this, maybe just as a bunch of sliders inside the Lighting Settings modal dialog. Even something as simple as 3 sliders, x y and z, directly mapping to Unity's transform coordinates. Doesn't have to be intuitive or easy, I would just really like to be _able_ to edit the angle at all.

Thanks for considering!

**Comment by emilianavt** [comment]
You can adjust the lighting with Ctrl + left click. Pressing "D" will reset it. The currently set light position can be saved in the "Light settings" window.

**Comment by pepper-lees** [comment]
> You can adjust the lighting with Ctrl + left click. Pressing "D" will reset it. The currently set light position can be saved in the "Light settings" window.

@emilianavt omg, thank you, I had no idea! FYI: I don't think this info is in the help text on https://www.vseeface.icu. Earlier, before I wrote this request, I did a search for every mention of the word 'light' and couldn't find any mention of how to change the lighting angle. The only related thing I could find was info on how to set up the unity editor to match VSeeFace's default lighting settings. Just now, I searched for 'ctrl' and still don't see it documented.
